<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>PlayerControls</class>
    <widget class="QWidget" name="PlayerControls">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>200</height>
            </rect>
        </property>
        <property name="styleSheet">
            <string notr="true">background-color: rgb(23, 24, 27);</string>
        </property>
        <layout class="QGridLayout" name="controlsLayout">
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item row="0" column="0">
                <layout class="QGridLayout">
                    <property name="spacing">
                        <number>0</number>
                    </property>
                    <item row="0" column="0">
                        <widget class="QLabel" name="stopButton">
                            <property name="styleSheet">
                                <string notr="true">color:rgb(255, 255, 255);font: 21pt &quot;Font Awesome 5 Free&quot;;</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="1">
                        <widget class="QLabel" name="backwardButton">
                            <property name="styleSheet">
                                <string notr="true">color:rgb(255, 255, 255);font: 21pt &quot;Font Awesome 5 Free&quot;;</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="2">
                        <widget class="QLabel" name="playPauseButton">
                            <property name="styleSheet">
                                <string notr="true">color:rgb(255, 255, 255);font: 21pt &quot;Font Awesome 5 Free&quot;;</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="3">
                        <widget class="QLabel" name="forwardButton">
                            <property name="styleSheet">
                                <string notr="true">color:rgb(255, 255, 255);font: 21pt &quot;Font Awesome 5 Free&quot;;</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="0" column="4">
                        <widget class="QLabel" name="moreButton">
                            <property name="styleSheet">
                                <string notr="true">color:rgb(255, 255, 255);font: 21pt &quot;Font Awesome 5 Free&quot;;</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="0">
                        <widget class="TimeLabel" name="time">
                            <property name="styleSheet">
                                <string notr="true">color: rgb(255, 255, 255);</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item row="1" column="4">
                        <widget class="TimeLabel" name="duration">
                            <property name="styleSheet">
                                <string notr="true">color: rgb(255, 255, 255);</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                            </property>
                        </widget>
                    </item>
                </layout>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>TimeLabel</class>
            <extends>QLabel</extends>
            <header>widgets/TimeLabel.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
